### PR TITLE
Fix: Update test reporter version

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] updates and asserts the test reporter version

### CODE REVIEW

File: src/Constants/Version.php [hunk 1]
```diff
@@ -12,5 +12,5 @@
      *
      * @var string
      */
-    const VERSION = '0.1.2';
+    const VERSION = '0.4.0';
```
Comment (localheinz) [inline src/Constants/Version.php]: This should probably be updated before tagging, right?

**Comment by pbrisbin** [comment]
Will save this PR for last to incorporate all the other work you've done in one version bump (if that's OK). In that case, should this go to `0.2`?

File: src/Constants/Version.php (hunk 1 above)
Comment (maxjacobson) [inline src/Constants/Version.php]: Whoops, I missed that when making https://github.com/codeclimate/php-test-reporter/pull/64 -- thanks for catching it

**Comment by localheinz** [comment]
Not sure how you guys proceed with the tagging, but as a last thing before creating a tag I would probably commit an update to the version and push to `master`, then tag it - what do you think?

**Comment by pbrisbin** [comment]
> I would probably commit an update to the version and push to master, then tag it

That's probably how I'd prefer to do it. Which means this PR is unnecessary and we'll handle it ourselves after all others have been merged, right?

/cc @maxjacobson

**Comment by localheinz** [comment]
@pbrisbin 

Sounds good! Closing this one then!

File: tests/Unit/VersionTest.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace CodeClimate\PhpTestReporter\Tests\Unit;
+
+use CodeClimate\PhpTestReporter\Constants\Version;
+
+final class VersionTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConstant()
+    {
+        $this->assertSame('0.4.0', Version::VERSION);
```
Comment (localheinz) [inline tests/Unit/VersionTest.php]: Ha, that is unless you want this test to exist, hehe!

**Comment by maxjacobson** [comment]
@pbrisbin I've taken to opening a PR for version-bumping commits, rather than pushing directly to master. The main reason is that we generally use the "protected branch" feature of GitHub, which doesn't allow me to push to master.

**Pull request closed without merging**